src/test/java/com/avispl/symphony/dal/communicator/aver/ptz/UDPCommunicatorTest.java

```diff
@@ -10,29 +10,21 @@
 import org.junit.Before;
 import org.junit.Test;
 
-import com.avispl.symphony.dal.communicator.aver.ptz.mock.UDPServer;
-
 /**
  * Unit test for UDP Communicator
  * Send and retrieve data success
  *
- * @author Harry
- * @version 1.0
- * @since 1.0
+ * @author Hieu.LeMinh
```

Comment:
@author Harry to make it consistent with the other? I think

Reply:
Yep, I will fix it